### PR TITLE
NAME case-sensitivity update

### PR DESCRIPTION
**Pull request by xiaobill8**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -195,6 +205,8 @@ Adds a contact to your task book.
 
 Format: `contact add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [#/TAG]…`
 
+Note: NAME is case-sensitive, as such `John Doe` will be treated as a different
```
Comment (seetohjinwei) [inline docs/UserGuide.md:208]: Perhaps this sentence is cut off?

**Review by seetohjinwei — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -154,7 +154,17 @@ The examples in this guide are formatted with the following conventions:
 
 </div>
 
-### Accepted Date Formats
+### Points to Note
+
+#### NAME Parameter
+
+Where the NAME parameter is required (e.g. `n/NAME`), specify the **exact name**
+- in full
```
Comment (dexter-sim) [inline docs/UserGuide.md:162]: @bananamonkey4655 also wrote this full name note somewhere else in the user guide. Perhaps we should decide on one place to put it?

Comment (xiaobill8) [reply docs/UserGuide.md]: Don't see it though, I've placed it at the bottom with the Accepted Date Formats

**Review by seetohjinwei — APPROVED** [review]
LGTM!

**Pull request merged**